esmvalcore/_data_finder.py

```diff
@@ -317,18 +462,21 @@ def get_output_file(variable, preproc_dir):
         _replace_tags(cfg['output_file'], variable)[0],
     )
     if variable['frequency'] != 'fx':
-        outfile += '_{start_year}-{end_year}'.format(**variable)
+        timerange = variable['timerange'].replace('/', '-')
+        outfile += f'_{timerange}'
+
     outfile += '.nc'
     return outfile
 
 
 def get_statistic_output_file(variable, preproc_dir):
     """Get multi model statistic filename depending on settings."""
+    variable['timerange'] = variable['timerange'].replace('/', '-')
```

Comment:
This results in a wrong `timerange` attribute for the multimodel statistics output file. Could you do it without modifying the `variable` dict?